pom.xml

```diff
@@ -1,625 +1,657 @@
-<project xmlns="http//maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
I had actually addressed this at the summit in Singapore, see https://github.com/openmrs/openmrs-standalone/pull/26